randovania/data/json_data/prime1.json

```diff
@@ -448,18 +563,18 @@
                     "nodes": [
                         {
                             "name": "Dock to Crater Tunnel A",
-                            "heal": false,
+                            "heal": true,
                             "coordinates": {
-                                "x": -9.72091293334961,
-                                "y": -29.89837646484375,
-                                "z": 2.500004884185728
+                                "x": -9.72,
+                                "y": -29.9,
+                                "z": 2.5
                             },
                             "node_type": "dock",
                             "dock_index": 0,
                             "connected_area_asset_id": 1238049635,
                             "connected_dock_index": 1,
                             "dock_type": 0,
-                            "dock_weakness_index": 0,
+                            "dock_weakness_index": 1,
```

Comment:
The dock_weakness_database has the same index for Ice/Wave/Plasma doors, so this is likely wrong :X